File: weezam-user/src/main/java/com/weezam/challenge/user/adapter/in/rest/dto/UserDto.java
package com.weezam.challenge.user.adapter.in.rest.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserDto {

    private Long id;
    private String firstName;
    private List<PaymentMethodDto> paymentMethods;
    private Double maxWithdrawalAmount;
    
}

File: src/main/java/com/wezaam/withdrawal/dto/UserDTO.kt
package com.wezaam.withdrawal.dto

import com.wezaam.withdrawal.model.PaymentMethod

data class UserDTO (
        val id:Long,
        val name:String,
        val paymentMethods: List<PaymentMethod?>? = null,
        var maxWithdrawalAmount: Double? = null )
